tests/console/python310_version_check.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: python310
+# Summary: Run python310 testsuite
+# - Test suitable only for SLE15SP4+
+# - Check that Python 3.6 is the main version installed
+# - Check that Python 3.10 is available to install
+# - Run some basic test (man_or_boy.py)
+# Maintainer: QE Core <qe-core@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use version_utils;
+use utils "zypper_call";
+
+sub run {
+    select_console("root-console");
```

Comment:
I think we are supposed to use `serial_terminal` in console based tests wherever possible.

Reply:
I've changed the execution to serial_console, as suggested. Thanks!